common/app/views/fragments/amp/adConsent.scala.html

```diff
@@ -24,31 +24,35 @@
    {
        "consents": {
            "adconsent": {
-               "promptIfUnknownForGeoGroup": "eea"
+               "promptIfUnknownForGeoGroup": "eea",
```

Comment:
This is a different selection criteria than the web, currently the banner and TCF use:

```
const isInEU = (): boolean =>
    (getCookie('GU_geo_continent') || 'OTHER').toUpperCase() === 'EU';
```

Reply:
GRPR seems to apply to EEA, a superset of the EU